Elf/__init__.py

```diff
@@ -879,6 +879,14 @@ def getPlatform(self):
                 desc0 = int(note.desc[0])
                 return osnotes.get(desc0, 'unknown')
 
+            #if note.name == b'01.01\x00\x00\x00':
+            #    return 'freebsd'
+
+        if self.getSection('.comment'):
```

Comment:
while testing this PR, failure to identify FreeBSD platforms became a clear problem.